<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>761</width>
    <height>417</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ticker Scraper</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0,0,0,0">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="btn_manager_toggle">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Start Task(s)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_add_event">
        <property name="styleSheet">
         <string notr="true">background:green;</string>
        </property>
        <property name="text">
         <string>Add Event</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_edit_event">
        <property name="styleSheet">
         <string notr="true">background:blue;</string>
        </property>
        <property name="text">
         <string>Edit Event</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_delete_event">
        <property name="styleSheet">
         <string notr="true">background:red;</string>
        </property>
        <property name="text">
         <string>Delete Event</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btn_settings">
        <property name="styleSheet">
         <string notr="true">background:gray;</string>
        </property>
        <property name="text">
         <string>Settings</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="QTableWidget" name="table_widget">
        <column>
         <property name="text">
          <string>Name</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Interval</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Max ticket cost</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Ignore Text</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>URL</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Added On</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Last Check</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
